recipe/meta.yaml

```
@@ -41,8 +42,9 @@ source:
     folder: third_party/thrust/dependencies/cub
 
 build:
-  skip: true  # [not linux]
-  number: 0
+  skip: true  # [win]
+  number: 1
+  skip: true  # [py<39]
```

Might be worth grouping the `skip`s together or possibly putting them all in one line like so

```suggestion
  skip: true  # [win or py<39]
```